includes/api/class-edd-api-v2.php

```diff
@@ -73,7 +73,7 @@ public function get_products( $args = array() ) {
 			}
 
 			if( ! empty( $args['category'] ) ) {
-				if ( strpos( $args['category'], ',' ) ) {
+				if ( is_string( $args[ 'categrory' ] ) ) {
 					$args['category'] = explode( ',', $args['category'] );
 				}
 
```

Comment:
Because we're always getting a `string` to `array` conversion here...the `is_numeric` check will always fail and is no longer necessary, essentially making it unused code. Since the `foreach` loop in the `is_array` check does check for `is_numeric` so that'll catch things.